lsp/expert.lua

```diff
@@ -3,10 +3,19 @@
 --- https://github.com/elixir-lang/expert
 ---
 --- Expert is the official language server implementation for the Elixir programming language.
+---
+--- 'root_dir' is chosen like this: if two or more directories containing `mix.exs` were found when searching directories upward, the second one (higher up) is chosen, with the assumption that it is the root of an umbrella app. Otherwise the directory containing the single mix.exs that was found is chosen.
```

Comment:
```suggestion
--- 'root_dir' is chosen like this: if two or more directories containing `mix.exs` were found when
--- searching directories upward, the second one (higher up) is chosen, with the assumption that it
--- is the root of an umbrella app. Otherwise the directory containing the single mix.exs that was
--- found is chosen.
```